tests/integration/index/create_get_test.go

```diff
@@ -23,7 +23,7 @@ func TestIndexGet_ShouldReturnListOfExistingIndexes(t *testing.T) {
 		Actions: []any{
```

Comment:
todo: (not necessarily in this file, this is just the first test file) Please add some integration tests testing implicit field level directives, the 'default' behaviour of the `direction` property, the overriding of a default `direction` by explicitly stating it in a field in `includes`, the overriding of the field location by explicitly stating a field when the directive is hosted on a field, and composite indexes declared on a field.

Reply:
done